recipe/conda_build_config.yaml

```
@@ -424,7 +424,9 @@ giflib:
 glew:
   - 2.1
 glib:
-  - 2.58
+  - 2.58  # [not (osx and arm64)]
+  # Can't update to 2.66 on osx-64 yet because that requires a newer macOS version than 10.9
```

Maybe use `x64` to make it clear that you don't mean `arm`.
```suggestion
  # Can't update to 2.66 on osx-x64 yet because that requires a newer macOS version than 10.9
```

---

`osx-64` is the conda subdir name for `x86_64`

---

```suggestion
  # Can't update to 2.66 on `osx and x86_64` yet because that requires a newer macOS version than 10.9
```

---

changed it to `osx and x86_64`. ready for review